app/components/ui/NewsArticles/NewsArticles.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from "react";
+import {
+  CarouselProvider,
+  Slider,
+  Slide,
+  ButtonBack,
+  ButtonNext,
+  DotGroup,
+} from "pure-react-carousel";
+
+import * as dataService from "../../../utils/DataService";
+import { NewsArticle } from "../../../models";
+
+import styles from "./NewsArticles.module.scss";
+
+const formatDate = (date: string): string =>
+  new Date(date).toLocaleDateString("en-US", {
+    year: "numeric",
+    month: "short",
+    day: "numeric",
+  });
+
+const ArticleItem = ({ article }: { article: NewsArticle }) => (
+  <div className={styles.articleItem}>
+    <div className={styles.newsContent}>
+      <p className={styles.articleTitle}>
+        {article.headline}
+        {article.effective_date && (
+          <>
+            {" "}
+            <span className={styles.dateSeperator}>|</span>{" "}
+            <span className={styles.articleDate}>
+              {formatDate(article.effective_date)}
+            </span>
+          </>
+        )}
+      </p>
+      <p className={styles.articleBody}>{article.body}</p>
+    </div>
+    {article.url && (
+      <a
+        href={article.url}
+        className={styles.articleLink}
+        target="_blank"
+        rel="noreferrer"
+      >
+        <span className={styles.linkText}>Read more →</span>
+        <span className={styles.linkText_mobile}>
+          More{" "}
+          <i className={`material-icons ${styles.mobileLinkChevron}`}>
+            keyboard_arrow_right
+          </i>
+        </span>
+      </a>
+    )}
+  </div>
+);
+
+export const NewsArticles = () => {
+  const [collapseCarousel, setCollapseCarousel] = useState(false);
+  const [breakingNewsArticles, setBreakingNewsArticles] = useState<
+    NewsArticle[]
+  >([]);
+
+  useEffect(() => {
+    const fetchBreakingNewsArticles = async () => {
+      const response = await dataService.get("/api/news_articles");
+      const { news_articles }: { news_articles: NewsArticle[] } = response;
+      setBreakingNewsArticles(news_articles);
+    };
+
+    fetchBreakingNewsArticles();
+  }, []);
+
+  if (!breakingNewsArticles || breakingNewsArticles.length === 0) return null;
+
+  return (
+    <div
+      className={`${styles.container} ${
+        collapseCarousel && styles.carouselCollapsed
```

Comment:
While looking at the dynamic CSS classes on the actual HTML, I noticed that this literally renders as `false` in the class attributes. I think you may need to turn this into a ternary expression where you explicitly return the empty string if the condition is false.

Reply:
Ahhh I hadn't thought of that -- good callout on that one